internal/install/recipe_installer.go

```diff
@@ -123,36 +130,57 @@ func (i *RecipeInstaller) Install() error {
 		}
 
 		for _, r := range recipes {
-			log.Debugf("Found available integration %s.", r.Name)
+			log.Debugf(fmt.Sprintf("Found available integration %s.", r.Name))
 		}
 
 		i.reportRecipesAvailable(recipes)
 	}
 
+	log.Debugf(fmt.Sprintf("AssumeYes:%t.", i.AssumeYes))
```

Comment:
This multi-statement debug block can also be achieved with `log.Debugf(%+v\n.", i.InstallerContext)`, but with slightly harder-to-read format.

Reply:
Nice, I was wishing for something like that, I'll update. I don't mind that it is harder to read in debug. It's also more maintainable if more fields are added in the future.

Reply:
Ho my bad, I've mistaken log.DebugLn with log.Debugf.
Do I need to explicit do a line return if I use log.Debugf ?